Formula/r.rb

```diff
@@ -0,0 +1,172 @@
+class RDownloadStrategy < SubversionDownloadStrategy
+  def stage
+    cp_r File.join(cached_location, "."), Dir.pwd
+  end
+end
+
+class R < Formula
+  desc "Software environment for statistical computing"
+  homepage "https://www.r-project.org/"
+  url "https://cran.rstudio.com/src/base/R-3/R-3.4.1.tar.gz"
+  sha256 "02b1135d15ea969a3582caeb95594a05e830a6debcdb5b85ed2d5836a6a3fc78"
+  head "https://svn.r-project.org/R/trunk", :using => RDownloadStrategy
+
+  bottle do
+    sha256 "d0254993416c177d7fa49b9cde95eb8bd262e3a801408b21951cc0f7755e0a0e" => :sierra
+    sha256 "2098376a2d552573a1b0e2ff29c076b05a0161ec276260b5b76a80e87d5cd6c1" => :el_capitan
+    sha256 "be31e78c3df77a46e91500b4809cb7f89bceacabc0c38d1bc3e56beab31bff6e" => :yosemite
+    sha256 "aa7dd6e0c0b7cd1bd5e1471c37a505cd69009e53d9e5c35a77975e547629662d" => :x86_64_linux
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "texinfo" => :build
+  depends_on :fortran
+  depends_on "readline"
+  depends_on "gettext"
+  depends_on "libtiff"
+  depends_on "pcre"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "xz"
+
+  depends_on "homebrew/science/openblas" => :optional
+  depends_on "pango" => :optional
+  depends_on :java => :optional
+
+  depends_on "cairo"
+
+  patch :DATA
+
+  def install
+    # Fix dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
+    if MacOS.version == "10.11" && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
+      ENV["ac_cv_have_decl_clock_gettime"] = "no"
+    end
+
+    # Fix cairo detection with Quartz-only cairo
+    inreplace ["configure", "m4/cairo.m4"], "cairo-xlib.h", "cairo.h"
+
+    args = [
+      "--prefix=#{prefix}",
+      "--with-libintl-prefix=#{Formula["gettext"].opt_prefix}",
+      "--enable-memory-profiling",
+      "--enable-R-framework",
+      "--with-cairo",
+      "--without-x",
+    ]
+
+    # don't remember Homebrew's sed shim
+    args << "SED=/usr/bin/sed"
+
+    # Disable building against the Aqua framework with CLT >= 6.0.
+    # See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=63651
```

Comment:
This issue last saw an update 2015-12-12 22:54:14 UTC, so might want fx's opinion on what to do here.